### PR TITLE
WCAG upgrade - Create test page for: Color (foreground/background)

### PR DESCRIPTION
**Pull request by BrahimMahadi**
This pull request includes a test page with working examples filled with lorem ipsum text for Utilities/helpers. More info can be found by referencing WET-319.

### CODE REVIEW

File: sites/footers/reports/a11y-1-en.hbs
Comment (duboisp) [inline sites/footers/reports/a11y-1-en.hbs:1]: The propose change for this file need to be put in another PR (if that not already the case)

File: sites/footers/reports/a11y-1-fr.hbs
Comment (duboisp) [inline sites/footers/reports/a11y-1-fr.hbs:1]: The propose change for this file need to be put in another PR (if that not already the case)

File: sites/footers/reports/a11y-1.json
Comment (duboisp) [inline sites/footers/reports/a11y-1.json:1]: The propose change for this file need to be put in another PR (if that not already the case)

**Review by duboisp — CHANGES_REQUESTED** [review]
Here a quick review

* [ ] The site footer ACR would need to be removed from this PR. The easiest way to make this change will be to use a local git client to make the changes.
* [ ] Missing the French content equivalent page
* [ ] Missing the index.json-ld which will contains the metadata for this common component
* [ ] I quickly saw that the heading structure for the "colour.html" page need to be adjusted like at line 62 where it should be an h5 instead of being an h4

File: sites/footers/index.json-ld
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply sites/footers/index.json-ld:1]: Removed footer files from this PR.

File: sites/footers/reports/a11y-1-en.hbs
Comment (BrahimMahadi) [reply sites/footers/reports/a11y-1-en.hbs:1]: Removed footer files from this PR.

File: sites/footers/reports/a11y-1.json
Comment (BrahimMahadi) [reply sites/footers/reports/a11y-1.json:1]: Removed footer files from this PR.

File: sites/footers/reports/a11y-1-fr.hbs
Comment (BrahimMahadi) [reply sites/footers/reports/a11y-1-fr.hbs:1]: Removed footer files from this PR.

**Pull request closed without merging**